### ISSUE TITLE
Broken indentation

### ISSUE THREAD

**Issue by myint** [google/yapf#10]
`yapf` seems to break the indentation of [Python 3.4's `bisect.py`](https://hg.python.org/cpython/file/3.4/Lib/bisect.py). Note that the lines after the `else:` should be indented one more level than what `yapf` is currently doing.

```
$ python -m yapf -d bisect.py
```
```diff
--- bisect.py	(original)
+++ bisect.py	(reformatted)
@@ -1,4 +1,5 @@
 """Bisection algorithms."""
+
 
 def insort_right(a, x, lo=0, hi=None):
     """Insert item x in list a, and keep it sorted assuming a is sorted.
@@ -14,12 +15,15 @@
     if hi is None:
         hi = len(a)
     while lo < hi:
-        mid = (lo+hi)//2
+        mid = (lo + hi) // 2
         if x < a[mid]: hi = mid
-        else: lo = mid+1
+        else:
+        lo = mid + 1
     a.insert(lo, x)
 
-insort = insort_right   # backward compatibility
+
+insort = insort_right # backward compatibility
+
 
 def bisect_right(a, x, lo=0, hi=None):
     """Return the index where to insert item x in list a, assuming a is sorted.
@@ -37,12 +41,15 @@
     if hi is None:
         hi = len(a)
     while lo < hi:
-        mid = (lo+hi)//2
+        mid = (lo + hi) // 2
         if x < a[mid]: hi = mid
-        else: lo = mid+1
+        else:
+        lo = mid + 1
     return lo
 
-bisect = bisect_right   # backward compatibility
+
+bisect = bisect_right # backward compatibility
+
 
 def insort_left(a, x, lo=0, hi=None):
     """Insert item x in list a, and keep it sorted assuming a is sorted.
@@ -58,9 +65,10 @@
     if hi is None:
         hi = len(a)
     while lo < hi:
-        mid = (lo+hi)//2
-        if a[mid] < x: lo = mid+1
-        else: hi = mid
+        mid = (lo + hi) // 2
+        if a[mid] < x: lo = mid + 1
+        else:
+        hi = mid
     a.insert(lo, x)
 
 
@@ -80,9 +88,10 @@
     if hi is None:
         hi = len(a)
     while lo < hi:
-        mid = (lo+hi)//2
-        if a[mid] < x: lo = mid+1
-        else: hi = mid
+        mid = (lo + hi) // 2
+        if a[mid] < x: lo = mid + 1
+        else:
+        hi = mid
     return lo
 
 # Overwrite above definitions with a fast C implementation
```

Here is a minimal example:

```python
if True: pass
else: pass
```
```diff
--- foo.py	(original)
+++ foo.py	(reformatted)
@@ -1,2 +1,3 @@
 if True: pass
-else: pass
+else:
+pass
```

Similarly:

```python
class Foo(object): pass
```
```diff
--- foo.py	(original)
+++ foo.py	(reformatted)
@@ -1 +1,2 @@
-class Foo(object): pass
+class Foo(object):
+pass
```

Thanks

**Comment by gwelymernans** [comment]
This should have been fixed by fc1da47. PTAL

**Comment by myint** [comment]
Looks good.
